mlflow/tracking/context/databricks_job_context.py

```diff
@@ -36,4 +39,7 @@ def tags(self):
             tags[MLFLOW_DATABRICKS_JOB_TYPE] = job_type
         if webapp_url is not None:
             tags[MLFLOW_DATABRICKS_WEBAPP_URL] = webapp_url
+        if workspace_url is not None:
+            tags[MLFLOW_DATABRICKS_WORKSPACE_URL] = workspace_url
+            tags[MLFLOW_DATABRICKS_WORKSPACE_ID] = workspace_id
```

Comment:
Nit: What if `workspace_url` is None but `workspace_id` is not None, and vice versa?

Reply:
Yes, it makes sense to not assume that they are dependent on each other.